pkg/preparer/orchestrate.go

```diff
@@ -56,6 +56,18 @@ func (p *Preparer) WatchForHooks(quit chan struct{}) {
 	}
 }
 
+// Sync the hooks just once. This is useful at preparer startup so that hooks
+// will be synced before normal pod management begins
+func (p *Preparer) SyncHooksOnce() {
+	hookErrCh := make(chan error)
```

Comment:
Probably a good idea to just return this `error` directly, since there will only be one opportunity